src/protocol_pro_2.cpp

```diff
@@ -41,7 +42,53 @@ Pro2ProtocolObject::Pro2ProtocolObject(
       std::thread([this]() { this->motors_control_loop(30); });
 }
 
-void Pro2ProtocolObject::update_drivetrim(double value) { trimvalue_ += value; }
+void Pro2ProtocolObject::get_params_file() {
+  // open yaml
+  std::ifstream robotconfig;
+  std::string const HOME = std::getenv("HOME") ? std::getenv("HOME") : ".";
+  robotconfig.open(HOME + "/rover.config");
```

Comment:
"/rover.config" should probably be defined in all caps in a header file. something like PERSISTENT_PARAM_FILE or similar.